api/internal/managers/infra/install.go

```diff
@@ -98,6 +99,15 @@ func (m *infraManager) install(ctx context.Context, rc runtimeconfig.RuntimeConf
 		return fmt.Errorf("parse license: %w", err)
 	}
 
+	var airgapInfo *kotsv1beta1.Airgap
+	if m.airgapBundle != "" {
+		var err error
+		airgapInfo, err = airgap.AirgapInfoFromPath(m.airgapBundle)
+		if err != nil {
+			return fmt.Errorf("failed to get airgap info: %w", err)
+		}
+	}
+
```

Comment:
Can you follow the same pattern described [here](https://github.com/replicatedhq/embedded-cluster/pull/2336#discussion_r2157646365) where you make airgapInfo a property of the manager and [pass](https://github.com/replicatedhq/embedded-cluster/blob/d52e8a9bdde268112b2c91d3fb43c17493e58ae1/api/controllers/install/controller.go#L194-L201) it with the function WithAirgapInfo?